### PR TITLE
Fix bug when tx was already imported by node

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: safe_relay_service/relay/tasks.py [hunk 1]
```diff
@@ -218,8 +218,11 @@ def deploy_safes_task(retry: bool=True) -> None:
                         safe_funding.safe_deployed_tx_hash = creation_tx_hash
                         safe_funding.save()
                 except TransactionAlreadyImported:
-                    logger.warning("Safe=%s transaction was already imported by the node")
-                    return
+                    logger.warning("Safe=%s transaction was already imported by the node", safe_address)
+                    safe_funding.safe_deployed_tx_hash = safe_creation.tx_hash
+                    safe_funding.safe_deployed = False
+                    safe_funding.save()
+                    continue
```
Comment (rmeissner) [inline safe_relay_service/relay/tasks.py]: Why do we use `continue` here, but not for the other cases?

Comment (Uxio0) [reply safe_relay_service/relay/tasks.py]: That's right, it doesn't need to be there :sweat_smile:

Comment (Uxio0) [reply safe_relay_service/relay/tasks.py]: I will remove in the next update as it doesn't hurt

File: safe_relay_service/relay/tasks.py [hunk 2]
```diff
@@ -218,8 +218,11 @@ def deploy_safes_task(retry: bool=True) -> None:
                         safe_funding.safe_deployed_tx_hash = creation_tx_hash
                         safe_funding.save()
                 except TransactionAlreadyImported:
-                    logger.warning("Safe=%s transaction was already imported by the node")
-                    return
+                    logger.warning("Safe=%s transaction was already imported by the node", safe_address)
+                    safe_funding.safe_deployed_tx_hash = safe_creation.tx_hash
+                    safe_funding.safe_deployed = False
```
Comment (rmeissner) [inline safe_relay_service/relay/tasks.py]: Do we have to explicitely set it to `False`

shouldn't it just stay what it is?

Comment (Uxio0) [reply safe_relay_service/relay/tasks.py]: I found some weird cases when safe is deployed but the tx_hash is empty, and wrote this to make sure that the state is what should be and no other instance is writing `safe_deployed`, but as Django writes all the fields of the object, you are right, it's no use

**Pull request merged**